docs/howto/garbage-collection/standalone-gc.md

```diff
@@ -298,6 +298,6 @@ run_id=<GC run id>
 # Download the CSV file
 aws s3 cp "$storage_ns/_lakefs/retention/gc/reports/$run_id/deleted.csv" "./run_id-$run_id.csv"
 
-# Move all addresses to the output bucket under the run_id prefix
-cat run_id-$run_id.csv | tail -n +2 | head -n 10 | xargs -I {} aws s3 mv "$storage_ns/{}" "$output_bucket/run_id=$run_id/"
```

Comment:
`head -n 10` was there for my local testing, it's taking only the first 10 addresses.